test/test_benchmark.py

```diff
@@ -871,98 +871,98 @@ def output_parser(output):
   def test_native_functions(self):
     def output_parser(output):
       return float(re.search(r'Total time: ([\d\.]+)', output).group(1))
-    self.do_benchmark('native_functions', read_file(test_file('benchmark_ffis.cpp')), 'Total time:',
+    self.do_benchmark('native_functions', read_file(test_file('benchmark', 'benchmark_ffis.cpp')), 'Total time:',
                       output_parser=output_parser,
                       # Not minimal because this uses functions in library_browsers.js
                       emcc_args=['-sMINIMAL_RUNTIME=0'],
-                      shared_args=['-DBUILD_FOR_SHELL', '-I' + TEST_ROOT])
+                      shared_args=['-DBUILD_FOR_SHELL', '-I' + test_file('benchmark')])
 
   # Benchmarks the synthetic performance of calling function pointers.
   @non_core
   def test_native_function_pointers(self):
     def output_parser(output):
       return float(re.search(r'Total time: ([\d\.]+)', output).group(1))
-    self.do_benchmark('native_functions', read_file(test_file('benchmark_ffis.cpp')), 'Total time:',
+    self.do_benchmark('native_functions', read_file(test_file('benchmark', 'benchmark_ffis.cpp')), 'Total time:',
```

Comment:
These days you can just do `test_file('benchmark/benchmark_ffis.cpp')`  (i.e. you can use forward slashes here.   python's pathlib will take care of making them backslashes on windows).  Here and elsewhere in this PR.